x-pack/test/api_integration/apis/metrics_ui/inventory_threshold_alert.ts

```diff
@@ -77,51 +80,248 @@ export default function ({ getService }: FtrProviderContext) {
+            timeUnit: 'm',
+            sourceId: 'default',
+            threshold: [100],
+            comparator: '>',
+            shouldFire: [false],
+            shouldWarn: [false],
+            isNoData: [false],
+            isError: false,
+            currentValue: 0.7703333333333333,
+          },
+        });
       });
-      expect(results).to.eql({
-        'host-01': {
-          metric: 'cpu',
-          timeSize: 1,
-          timeUnit: 'm',
-          sourceId: 'default',
-          threshold: [100],
-          comparator: '>',
-          shouldFire: [true],
-          shouldWarn: [false],
-          isNoData: [false],
-          isError: false,
-          currentValue: 1.01,
-        },
+      it('should work FOR LAST 5 minute', async () => {
+        const options = {
+          ...baseOptions,
+          condition: { ...baseCondition, timeSize: 5 },
+          esClient: convertToKibanaClient(esClient),
+        };
+        const results = await evaluateCondition(options);
+        expect(results).to.eql({
+          'host-0': {
+            metric: 'cpu',
+            timeSize: 5,
+            timeUnit: 'm',
+            sourceId: 'default',
+            threshold: [100],
+            comparator: '>',
+            shouldFire: [true],
+            shouldWarn: [false],
+            isNoData: [false],
+            isError: false,
+            currentValue: 1.0376666666666665,
+          },
+          'host-1': {
+            metric: 'cpu',
+            timeSize: 5,
+            timeUnit: 'm',
+            sourceId: 'default',
+            threshold: [100],
+            comparator: '>',
+            shouldFire: [false],
+            shouldWarn: [false],
+            isNoData: [false],
+            isError: false,
+            currentValue: 0.9192,
+          },
+        });
       });
     });
-    it('should work FOR LAST 5 minute', async () => {
-      const options = {
-        ...baseOptions,
-        condition: { ...baseCondition, timeSize: 5 },
-        esClient: convertToKibanaClient(esClient),
-      };
-      const results = await evaluateCondition(options);
-      expect(results).to.eql({
-        'host-01': {
-          metric: 'cpu',
-          timeSize: 5,
-          timeUnit: 'm',
-          sourceId: 'default',
-          threshold: [100],
-          comparator: '>',
-          shouldFire: [false],
-          shouldWarn: [false],
-          isNoData: [false],
-          isError: false,
-          currentValue: 0.24000000000000002,
-        },
+
+    describe('Inbound network traffic per host', () => {
+      it('should work FOR LAST 1 minute', async () => {
+        const results = await evaluateCondition({
+          ...baseOptions,
+          condition: {
+            ...baseCondition,
+            metric: 'rx',
+            threshold: [1],
+          },
+          esClient: convertToKibanaClient(esClient),
+        });
+        expect(results).to.eql({
+          'host-0': {
+            metric: 'rx',
```

Comment:
@simianhacker Interesting. We call the inbound traffic `rx`? What does it stand for?

Reply:
rx = receive
tx = transmit